### PR TITLE
use json schemas to validate parse classes

### PR DESCRIPTION
**Pull request by ntraut**

### CODE REVIEW

File: src/parse/cloud/main.js [hunk 1]
```diff
@@ -49,5 +49,8 @@ module.exports = (Parse, event) => {
       }
       logger.info(`Parse correctly init`);
     })
-    .catch((err) => logger.info(`Issue to init Parse : ${err}`));
+    .catch((err) => {
+      logger.error(`Issue to init Parse : ${err}`);
+      throw err;
```
Comment (r03ert0) [inline src/parse/cloud/main.js:54]: the recommended way of throwing an error (eslint) is `throw new Error(err)`

Comment (ntraut) [reply src/parse/cloud/main.js:54]: Are you refering to the [no-throw-literal](https://eslint.org/docs/rules/no-throw-literal) rule? This rule is not enabled in the current project, but in this case `err` is already an error object (i throw a string in the code i use to validate the json in src/parse/cloud/setBeforeSave.js).

**Comment by ntraut** [comment]
For an unknown reason jest tests are not executed in github actions in the `ubuntu-latest` environment since they switched to `ubuntu-20.04`, but they are still executed with `ubuntu-18.04`.

**Comment by ntraut** [comment]
Judging from what I found there: https://stackoverflow.com/questions/63591711/jest-tests-not-running-with-jest-mongodb-preset, the problem with Ubuntu 20 comes that jest-mongodb works only with libcurl3 but ubuntu 20 has libcurl4.
So the problem comes from jest-mongodb which fails because of a missing library but without printing anything or returning an error code.

**Comment by r03ert0** [comment]
well spotted!

**Pull request merged**